.circleci/config.yml

```diff
@@ -230,6 +230,17 @@ jobs:
     environment:
       BROWSER: << parameters.browser >>
       BVER: << parameters.bver >>
+    trigger-qe-tests:
+      docker:
+        - image: circleci/node:latest
+      steps:
+        - run:
+            name: Trigger QE tests
+            command: |
+              curl --silent --output /dev/null -u USER_TOKEN: -X POST \
+              --header "Content-Type: application/json" \
+              -d '{"branch":"v${CIRCLE_TAG:0:1}","parameters":{"sdk_version": CIRCLE_TAG,"is_rc":true}}' \
```

Comment:
I have not experimented using circleci inbuilt variables in commands - but don't you need `$CIRCLE_TAG` instead of `CIRCLE_TAG` ?

Reply:
how does `"branch":"v${CIRCLE_TAG:0:1}` work - I assume it will resolve to` "branch": "v1"` for the tag `1.20.1-rc1` but we do not have a branched named like that.

Reply:
@makarandp0 For every major version QE has a corresponding slave branch version. This command should trigger workflow on that job.